.github/CODEOWNERS

```diff
@@ -1,2 +1,5 @@
 # Global code owners
-*   @martinsaporiti @javip97 @x1m3
\ No newline at end of file
+*   @martinsaporiti @javip97 @x1m3
+
+# UI code owners
+/ui @adrifdez @cneuro @amonsosanz @elias-garcia
```

Comment:
Shouldn't this be managed via teams in [Settings](https://github.com/0xPolygonID/sh-id-platform/settings/access)?

![Screenshot from 2023-03-01 17-07-39](https://user-images.githubusercontent.com/2691584/222195997-f91d2233-5d15-48e5-9328-84ba810a62c2.png)

Reply:
I agree, we could use groups. But there are people in the teams that maybe are not interested in being a code owner. I think raul jareño is in fe team. 
Let's discuss it or change it later to adapt to our needs.

Reply:
Repo permissions have changed and I cannot see right now who is in which group

Reply:
Agreed, let's involve @mmilenkovic  in this to understand how to manage this going forward. First step is actually to remove Raul from the FE team, as this no longer makes sense.